payload/payload.cc

```diff
@@ -15,12 +15,12 @@ void createInitialGlobalState(core::GlobalState &gs, const realmain::options::Op
         return;
     }
 
-    if (GLOBAL_STATE_PAYLOAD == nullptr) {
+    if (PAYLOAD_EMPTY) {
```

Comment:
ooc, why an extra boolean instead of checking `== nullptr` three times?

Reply:
It seemed convenient, given that we know that if we're using the binary payload we should consider all of them. Would you prefer the three checks?

Reply:
either way.